.versionrc.json

```diff
@@ -1,9 +1,3 @@
 {
-  "changelogHeader": "# Changelog\n\nThis document maintains a list of released versions and changes introduced by those versions.\nThis project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)\n",
-  "skip": {
-    "bump": true,
-    "changelog": false,
-    "commit": true,
-    "tag": true
```

Comment:
noticing both scripts are skipping commit and tag. Could we keep those two here under the global config?

Reply:
I had the same thought, but sided on keeping 'skips' together instead of having to look in two places. Separating them out into commands like you suggested should clean this up, methinks.